axelor-account/src/main/resources/domains/AccountChart.xml

```diff
@@ -11,6 +11,8 @@
     <string name="countryCode" title="Country code"/>
     <many-to-one name="metaFile" ref="com.axelor.meta.db.MetaFile" title="File"/>
     <many-to-many name="companySet" ref="com.axelor.apps.base.db.Company"/>
+    <integer name="accountCodeMinNbrChar" title="Account code min. nbr. char"
```

Comment:
accountCodeMinNbrChar --> accountCodeMinNbrCharSelect to follow the naming convention